abdulfatai360/banka
# #165483464 Implement get specific account on accounts resource

#### What does this PR do?
Add API for getting a specific account details

#### Description of Task to be completed?
- Have `GET /accounts/<account-number>/` endpoint working
- Write tests for the API's execution paths
- Have all the tests pass

#### How should this be manually tested?
On Terminal:
- Run `git clone https://github.com/abdulfatai360/banka.git` to clone this repo onto your local machine
- Navigate to the repo directory by running `cd banka/`
- Ensure your working directory is clean by running  `git status`  command
- Get a copy of the PR by typing  `git fetch origin pull/48/head:ft-view-account-detail-165483464`
- Now that you have a copy of the branch, switch to it using  `git checkout ft-view-account-detail-165483464`
- Your directory will now be an exact copy of the PR
- Run `npm install` to install all project's dependencies onto your local machine
- Type `npm run build` to run the build script and populate the `dist/` directory containing transpiled code
- Run `npm start` command to start the development server

#### What are the relevant pivotal tracker stories?
[#165483464](https://www.pivotaltracker.com/story/show/165483464)
